### PR TITLE
[KYUUBI #4379][BUILD] Exclude python virtual environments in the spotless maven plugin

### PR DESCRIPTION
**Pull request by a49a**
### _Why are the changes needed?_

Exclude python virtual environments in the spotless maven plugin

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2074,6 +2074,9 @@
                                 <include>*/README.md</include>
                                 <include>docker/playground/README.md</include>
                             </includes>
+                            <excludes>
+                                <exclude>docs/kyuubi/**/*.md</exclude>
```
Comment (pan3793) [inline pom.xml:2078]: are you supposing that the virtual env is named "kyuubi"?

Comment (a49a) [reply pom.xml]: I have changed the path to be more general.

File: pom.xml [hunk 2]
```diff
@@ -2074,6 +2074,9 @@
                                 <include>*/README.md</include>
                                 <include>docker/playground/README.md</include>
                             </includes>
+                            <excludes>
+                                <exclude>docs/*/lib/python*/**/*.md</exclude>
+                            </excludes>
```
Comment (pan3793) [inline pom.xml:2079]: what will happen if both `includes` and `excludes` is configured? cc @bowenliang123

Comment (bowenliang123) [reply pom.xml:2079]: https://github.com/diffplug/spotless/blob/maven/2.30.0/plugin-maven/src/main/java/com/diffplug/spotless/maven/AbstractSpotlessMojo.java#L284

**Review by bowenliang123 — APPROVED** [review]
LGTM

**Pull request closed without merging**